2-promisses/http-2-promise.ts

```diff
@@ -7,21 +7,44 @@ let params = {
     path: '/api/users'
 }
 
-function httpRequst(): Promise<string> {
-    return new Promise(function (resolve, reject) {
-        var req = https.request(params, function (res) {
 
-            var body: string[] = [];
-            res.on('data', function (chunk) {
-                body.push(chunk);
+var req = https.request(params, function (res) {
```

Comment:
I don't see the code you showed us on a call. This is the version where you changed the whole function.